session/rest.go

```diff
@@ -197,8 +196,8 @@ func makeHTTPRequest(session *Session, path string, requestType string, requestB
 	req.URL.RawQuery = encodeQuery(options)
 
 	if session.Debug {
-		log.Println("[DEBUG] Request URL: ", requestType, req.URL)
-		log.Println("[DEBUG] Parameters: ", requestBody.String())
```

Comment:
I'm thinking that you could do this with even less changes needed throughout:
```go
log := session.Logger
```

Similar changes are needed in session/xmlrpc.go